src/kixi/data_acquisition/db/inmemory.clj

```diff
@@ -0,0 +1,40 @@
+(ns kixi.data-acquisition.db.inmemory
```

Comment:
I try and use them as a way to define the interface I want to use. For example, I _shouldn't care_ how the DB implementation re-formats my table names as long as I can continue to refer to them in an idiomatic way (e.g. `:table-name` would need to be `"table_name"` in C* but is fine in InMemory version)